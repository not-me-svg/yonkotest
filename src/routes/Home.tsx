import styled from "styled-components";
import { Yonko3D } from "../components/Yonko3D";
import { BottomContent } from "../components/BottomContent";

export default function Home() {
  return (
    <>
      <HomeWrapper className="bg-color">
          <div className="h-100vh">
            <Yonko3D />
          </div>

          <div className="bg-color">
              <BottomContent></BottomContent>
          </div>
      </HomeWrapper>
    </>
  );
}

const HomeWrapper = styled.div`
  height: 100vh;
  position: relative;
`;
