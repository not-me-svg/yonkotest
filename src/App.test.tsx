// import React from 'react';
// import { render, screen } from '@testing-library/react';
// import App from './App';
// import Home from './routes/Home';
import React from 'react';
import { shallow } from 'enzyme';
import Home from './routes/Home';

describe('routes using memory router', () => {

  test('Should render component', () => {
    const wrapper = shallow(<Home /> );
    expect( wrapper ).toMatchSnapshot();
  });

  test('Should render tori gate', () => {
    const wrapper = shallow(<Home /> );
    const toriGate = wrapper.find('Yonko3D');
    expect( typeof(toriGate) ).toBe( 'object' );
  })
})