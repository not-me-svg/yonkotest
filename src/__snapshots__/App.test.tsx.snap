// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`routes using memory router Should render component 1`] = `ShallowWrapper {}`;
